frontend/src/app/modules/availability/new-slot/schedule-definition/schedule-definition.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+    <div class="schedule-container">
+        <div class="schedule-select-container">
+            <p>Schedule</p>
+            <div class="input">
```

Comment:
Probably this div is redundant

Reply:
I don't think so cause we have styles for it, this component is just a part of what we had in schedule component beforehand